index.html

```diff
@@ -609,8 +634,18 @@ <h2>Queue a <code>PerformanceEntry</code></h2>
             </li>
           </ol>
         </li>
-        <li>If the <var>add to performance entry buffer</var> flag is set, add
-        <var>new entry</var> to the <a>performance entry buffer</a>.
+        <li>
+          Determine if you should add <var>new entry</var> to a
```

Comment:
You should think of this algorithm as you think of code. You wouldn't tell a computer to "Determine if you should add", as that is not defined. What you can do here is:
a) Add that as a note, which is an equivalent to a code comment
b) Split out this sub-algorithm to a "helper function". You can add a separate algorithm below that defines what it means to "determine if you should add" and call it from here.

Does that make sense?

Reply:
I am not sure that the note is necessary.  I struggled with how to invoke it before, let me know if this makes more sense with the algorithm.